packages/cli/src/commands/secrets/command.js

```diff
@@ -2,9 +2,14 @@ import { packageName, getCommandName } from '../../util/pkg-name';
 
 export const secretsCommand = {
   name: 'secrets',
-  description: `NOTE: The ${getCommandName(
-    'env'
-  )} command is recommended instead of ${getCommandName('secrets')}`,
+  description:
+    'WARNING: On May 1st, 2024 secrets will be automatically converted to sensitive Environment Variables for Preview and Production environments. Secrets attached to Development environments will not be migrated.\n' +
+    `The ${getCommandName(
+      'secrets'
+    )} command will be deprecated at this time, please use the ${getCommandName(
+      'env'
+    )} command instead.\n` +
+    'See https://vercel.com/changelog/legacy-environment-variable-secrets-are-being-sunset for more information.',
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** use vercel.link

In this or a follow-up PR: Can we change this out for a vercel.link that points there instead? We typically use these from Vercel CLI messages for tracking and optional redirecting later. ([Examples](https://github.com/search?q=repo%3Avercel%2Fvercel%20vercel.link&type=code))

---

For reference: The vercel.link URLs are managed in bit.ly via Okta.‏